metricbeat/module/docker/network/_meta/fields.yml

```diff
@@ -33,8 +34,55 @@
             Total number of incoming packets.
     - name: out
       type: group
+      deprecated: true
       description: >
-        Outgoing network stats.
+        Outgoing network stats per second.
+      fields:
+        - name: bytes
+          type: long
+          format: bytes
+          description: >
+            Total number of outgoing bytes.
+        - name: dropped
+          type: scaled_float
+          description: >
+            Total number of dropped outgoing packets.
+        - name: errors
+          type: long
+          description: >
+            Total errors on outgoing packets.
+        - name: packets
+          type: long
+          description: >
+            Total number of outgoing packets.
+    - name: inbound
```

Comment:
To follow ECS, should it be on the root level? Meaning having it under `network.inbound.bytes` ?

Reply:
Umm, you are right, I was misinterpreting ECS for this case, would it be ok to have this without the `docker` preffix?

Reply:
If the content of the field matches ECS I would say yes.